src/CascadeSoftDeletes.php

```diff
@@ -41,7 +41,12 @@ protected static function bootCascadeSoftDeletes()
                     $model->{$relationship}->{$delete}();
                 } else {
                     foreach ($model->{$relationship} as $child) {
-                        $child->{$delete}();
+                        if ($child->pivot){
+                            $child->pivot->delete();
```

Comment:
You should use the `$delete` variable here, in order to ensure you perform a `forceDelete` if that is what the user is running.

`$child->pivot->{$method}();`